src/NuGet.Clients/NuGet.Tools/SolutionUserOptions.cs

```diff
@@ -22,22 +20,19 @@ namespace NuGetVSExtension
     [PartCreationPolicy(CreationPolicy.Shared)]
     internal sealed class SolutionUserOptions : IUserSettingsManager, IVsPersistSolutionOpts
     {
-        private const string NuGetOptionsStreamKey = "nuget";
+        private const string _nuGetOptionsStreamKey = "nuget";
```

Comment:
I never figured out why this is showing up differently on my machine. It seems like the right editor config is loaded.

@NuGet/nuget-client 
Do you get an editorconfig warning here?